### PR TITLE
fix: html

### PR DESCRIPTION
**Pull request by zirbest**
| Before | After |
| ------------- | ------------- |
| Screenshot from 2022-11-02 15 38 38 | Screenshot from 2022-11-02 15 37 02   |

| tree-sitter | regex |
| ------------- | ------------- |
|  [x] html   | [x] html |
|  [x] tsx     | [ ] tsx |

### CODE REVIEW

**Review by Mofiqul — APPROVED** [review]

File: lua/dracula/groups.lua [hunk 1]
```diff
@@ -198,8 +199,8 @@ local function setup(configs)
       htmlLink = { fg = colors.purple, underline = true, },
       htmlSpecialChar = { fg = colors.yellow, },
       htmlSpecialTagName = { fg = colors.cyan, },
-      htmlTag = { fg = colors.pink, },
-      htmlTagN = { fg = colors.pink, },
+      htmlTag = { fg = colors.cyan, },
```
Comment (mgerb) [inline lua/dracula/groups.lua:202]: [According to the Dracula Syntax Highlighting Specification](https://spec.draculatheme.com/#HtmlTags) I believe this should stay `pink`.

Comment (NoahTheDuke) [reply lua/dracula/groups.lua:202]: Not saying you shouldn't look at that, but this repo doesn't conform to that at all lol. For example, the spec says functions should be green, but they're defined as yellow here.

Comment (NoahTheDuke) [reply lua/dracula/groups.lua:202]: (I personally find the spec relies on pink for way too much, but I've also redefined nearly every group and mostly use this as a way to predefine the colors.)

Comment (mgerb) [reply lua/dracula/groups.lua:202]: Functions are green in the Treesitter groups here, but it seems there is an inconsistency with the standard groups. I'd assume the goal of this project would be to conform to the spec as closely as possible while still allowing overrides in the `setup`.

Comment (zirbest) [reply lua/dracula/groups.lua:202]: @mgerb ok :100: make a new PR

**Pull request merged**